bin/asstatd.pl

```diff
@@ -591,6 +591,42 @@ sub parse_sflow {
 			}
 		}
 		
+		# Extract src & dst IP from packet's header
+		my $srcip = undef;
+		my $dstip= undef;
+		my (undef, $ethertype, $ipdata) = unpack('a12H4a*', $sFlowSample->{'HeaderBin'});
+		if($ethertype eq 8100){
```

Comment:
Is that a typo maybe (no quotes)? Other than that, the patch looks good, thanks!